SwiftIntro.playground/Contents.swift

```diff
@@ -68,14 +120,15 @@ struct RunApp {
                 isPlaying: false
             )
         ]
-    }
+    }()
 }
 
-var runApp = RunApp(id: "1", startTime: Date())
-//print(runApp.elapsedTime())
-//
-//print(runApp.loadSongs().first!.title)
-//print(runApp.loadSongs().first!.authorName)
+var runApp = RunApp(id: "1", startTime: Date(), currentSong: Song(title: "TTT", authorName: "BBB", isPlaying: true))
```

Comment:
We didn't have enough time for our formatting lecture, but there is a nice rule of 3 or more parameters when calling a function. 
Don't you think that this code is more readable?

```suggestion
var runApp = RunApp(
    id: "1",
    startTime: Date(),
    currentSong:
        Song(
            title: "TTT",
            authorName: "BBB",
            isPlaying: true
        )
)
```